datamodel/initdb.d/01_create_dcsa_owner.sql

```diff
@@ -1,3 +1,4 @@
 \set ON_ERROR_STOP true
---CREATE USER dcsa_db_owner;
+DROP DATABASE IF EXISTS dcsa_openapi;
+DROP USER dcsa_db_owner;
```

Comment:
This will fail on an empty database (where the user does not already exist).

```suggestion
DROP USER IF EXISTS dcsa_db_owner;
```

This may still fail if it has any grants/permissions.  These are a bit harder to deal with according to https://stackoverflow.com/questions/40862077/drop-user-with-privileges-if-exist-in-postgres so I have left that out for now.